gui_agents/service/api_models.py

```
@@ -0,0 +1,136 @@
+    backend: str = Backend.LYBIC.value
+    mode: str = AgentMode.NORMAL.value
+    max_steps: int = 50
+    enable_takeover: bool = False
+    enable_search: bool = True
+    timeout: int = 3600  # 1 hour default timeout
+    config: Optional[Dict[str, Any]] = None
+    
+    def __post_init__(self):
+        """Validate request parameters"""
+        if self.max_steps <= 0:
+            raise ValueError("max_steps must be positive")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)